lib/IDE/CodeCompletion.cpp

```diff
@@ -4364,9 +4400,11 @@ class CompletionOverrideLookup : public swift::VisibleDeclConsumer {
     if (D->shouldHideFromEditor())
       return;
 
-    if (D->isFinal() ||
-        // A 'class' member with an initial value cannot be overriden either.
-        (D->isStatic() && D->getAttrs().hasAttribute<HasInitialValueAttr>()))
+    if (D->isFinal())
+      return;
+
+    // A 'class' member with an initial value cannot be overriden either.
```

Comment:
I'm not sure I understand. Can you give an example? Normally stored properties can be overridden with computed properties as long as they're not final.

Reply:
I believe this meant something like:
```swift
class C {
  class var foo = 1
}
```
But that's compile error so ... not sure.
Here, I'm just changing `D->getAttrs().hasAttribute<HasInitialValueAttr>()` to `isa<VarDecl>(D) && cast<VarDecl>(D)->hasInitialValue()`

Reply:
My point was that the presence of an initial value expression does not have any bearing on whether the declaration can be overridden or not. The comment above is incorrect.

I think the only relevant check here is whether the property is final (`static` is equivalent to `class final`).

Reply:
Right, I just removed this check.